bundles/binding/org.openhab.binding.fritzboxtr064/src/main/java/org/openhab/binding/fritzboxtr064/internal/AbstractItemMap.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.fritzboxtr064.internal;
+
+/***
+ * Abstract base implementation of an {@link ItemMap}.
+ *
+ * @author gitbock
```

Comment:
I refactored the ItemMap into the AbstractItemMap, so most of the code is still from gitbock. Since I made so many changes I now changed the author to me. After all, if I broke it, I should take the blame...